src/world/Spell/SpellAuras.cpp

```diff
@@ -1805,22 +1802,22 @@ void Aura::SpellAuraPeriodicDamage(bool apply)
         SetNegative();
         if (m_spellInfo->custom_BGR_one_buff_on_target & SPELL_TYPE_WARLOCK_IMMOLATE)
         {
-            m_target->SetFlag(UNIT_FIELD_AURASTATE, AURASTATE_FLAG_IMMOLATE);
+            m_target->addAuraState(AURASTATE_FLAG_CONFLAGRATE);
         }
```

Comment:
Please explain these flag changes:

- from immolate to conflagrate
- from poison to envenom
- from rejuvenate to swiftmend

Reply:
I will add definitions for them in AuraStates.h